Plugin.BluetoothLE.Uwp/GattService.cs

```diff
@@ -38,5 +39,22 @@ public override IObservable<IGattCharacteristic> WhenCharacteristicDiscovered()
 
             return this.characteristicOb;
         }
+
```

Comment:
Adding this function improved performance to the level necessary to prevent timeouts when connecting to MetaWearC.